### ISSUE TITLE
维度匹配问题

### ISSUE THREAD

**Issue by BethWu** [JusperLee/Conv-TasNet#55]
我在separation.py文件中加入了pesq的评估代码，后来发现
for key, egs in tqdm.tqdm(self.mix):
                #self.logger.info("Compute on utterance {}...".format(key))
                egs=egs.to(self.device)
                norm = torch.norm(egs,float('inf')).cpu()
                if len(self.gpuid) != 0:
                    print(egs.numel())
                    ests=self.net(egs)
                    print(ests[0].numel())
                    spks=[torch.squeeze(s.detach().cpu()) for s in ests]
中egs和ests的维度并不相同，于是我转而运行了Conv_TasNet.py文件
def test_convtasnet():
    x = torch.randn(320)
    nnet = ConvTasNet()
    s = nnet(x)
    print(str(check_parameters(nnet))+' Mb')
    print(s[0].shape)
    print(s[1].shape)
当我将x的元素限制在16000（即2s）内或32000（4s）以上时，生成的增强语音s的元素数量是一致的；而当我将x的元素设置为16000-32000之间时，s.shape就会比x.shape小，而且少的数量是不固定的。不清楚是什么原因？期盼得到博主的回复！
